### ISSUE TITLE
Typescript generation does not use type only import syntax

### ISSUE THREAD

**Issue by Swoorup** [anymaniax/orval#387]
## What are the steps to reproduce this issue?
Generate any typescript api: svelte, react or vue

## What happens?
Generated import: 
```typescript
import axios,{
  AxiosRequestConfig,
  AxiosResponse,
  AxiosError
} from 'axios'
import {
  useQuery,
  useMutation,
  UseQueryOptions,
  UseMutationOptions,
  QueryFunction,
  MutationFunction,
  UseQueryStoreResult,
  QueryKey
} from '@sveltestack/svelte-query'
```

## What were you expecting to happen?
Should instead generate: 

```typescript
import axios,{
  type AxiosRequestConfig,
  type AxiosResponse,
  AxiosError
} from 'axios'
import {
  useQuery,
  useMutation,
  type UseQueryOptions,
  type UseMutationOptions,
  type QueryFunction,
  type MutationFunction,
  type UseQueryStoreResult,
  type QueryKey
} from '@sveltestack/svelte-query'
```

## Any logs, error output, etc?
none

## Any other comments?
Since this feature was released in Typescript 3.8 (released Feb 20, 2020) I assume it should be safe to turn this on by default. Codebases with 'preserveValueImports' and 'isolatedModules' turned on will have issues otherwise like follows:

```
'UseMutationOptions' is a type and must be imported using a type-only import when 'preserveValueImports' and 'isolatedModules' are both enabled.ts(1444)
```

## What versions are you using?
**Operating System:** OSX
**Package Version:** v6.7.1
**Browser Version:** n/a

**Comment by anymaniax** [comment]
Hello @Swoorup, I am already using the import type from version 3.8. Your syntax was introduced in version 4.5. You can find the following explanation [here](https://devblogs.microsoft.com/typescript/announcing-typescript-4-5/#type-on-import-names). I prefer to wait a bit more until everyone has the time to move to version 4.5.


This works, but it would be nice to avoid two import statements for the same module. That’s part of why TypeScript 4.5 allows a type modifier on individual named imports so that you can mix and match as needed.

```ts
import { someFunc, type BaseType } from "./some-module.js";

export class Thing implements BaseType {
    someMethod() {
        someFunc();
    }
}
```

**Comment by Swoorup** [comment]
Hmm, so afaik, It currently needs to generate the following to make it back/forwards compatible?

```typescript
import {
  useQuery,
  useMutation
} from '@sveltestack/svelte-query';

import {
  UseQueryOptions,
  UseMutationOptions,
  QueryFunction,
  MutationFunction,
  UseQueryStoreResult,
  QueryKey
} from '@sveltestack/svelte-query';
```

I am currently getting this error using orval on sveltekit project.
```shell
'UseMutationOptions' is a type and must be imported using a type-only import when 'preserveValueImports' and 'isolatedModules' are both enabled.ts(1444)
```

**Comment by Swoorup** [comment]
Hmm, so @anymaniax, am I correct that it currently needs to generate the following to make it back/forwards compatible?

```typescript
import {
  useQuery,
  useMutation
} from '@sveltestack/svelte-query';

import {
  UseQueryOptions,
  UseMutationOptions,
  QueryFunction,
  MutationFunction,
  UseQueryStoreResult,
  QueryKey
} from '@sveltestack/svelte-query';
```

I am currently getting this error using orval on sveltekit project which uses the latest ts.
```shell
'UseMutationOptions' is a type and must be imported using a type-only import when 'preserveValueImports' and 'isolatedModules' are both enabled.ts(1444)
```

**Comment by anymaniax** [comment]
I thought it wasn't needed to split them if you have type and normal import at the same time.

**Comment by anymaniax** [comment]
Will be fix with the next release

**Comment by anymaniax** [comment]
Released with 6.9.0
